internal/tool/tool.go

```diff
@@ -256,11 +289,6 @@ func validateExecutionConfiguration(toolExecution codacy.ToolExecution) error {
 		return &ToolError{msg: fmt.Sprintf("Failed to configure Codacy Trivy: configured patterns don't match existing rules (provided %v)", patternIDs)}
 	}
 
-	if toolExecution.Files == nil || len(*toolExecution.Files) == 0 {
```

Comment:
If for some reason there are no files to analyse in the tool execution configuration, we still generate the SBOM.

Vulnerability and secret scanning will not produce results in this case.